### DISCUSSION TITLE
hidpi on mac

### DISCUSSION THREAD

**Discussion by ylca0** [WerWolv/ImHex#1952] [Feedback]
The rendering seems to have not implemented high-resolution display, the font is very blurry, and I find it quite tiring to view on the 4K 24-inch external monitor connected to my MacBook.
![image](https://github.com/user-attachments/assets/cbd4f66a-ef2a-44a4-94bc-411b49b993ee)

**Comment by WerWolv** [comment]
Hidpi has actually been implemented a long time ago already, however text was still not rendered at the full possible resolution.

This has been fixed now with one of the latest commits :)
![image](https://github.com/user-attachments/assets/e20ce770-f0cd-4039-a1d6-3205288bbc12)

**Reply by ylca0** [reply]
I am very happy to receive your reply, and I am very grateful for everything you have done!
